src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,4621 @@
+            "SENTBACK",
+            "SUBMITTED"
+          ]
+        },
+        "name": {
+          "type": "string",
+          "description": "The approval status of the Request in the current user's language",
+          "minLength": 1,
+          "maxLength": 2147483647
+        }
+      }
+    },
+    "AuditTrailAction": {
+      "type": "object",
+      "properties": {
+        "comment": {
+          "type": "string"
+        }
+      }
+    },
+    "BodyOfImportItinProposalAction": {
+      "type": "object",
+      "properties": {
+        "action": {
+          "$ref": "#/definitions/ActionOfImportItinProposalAction"
+        }
+      }
+    },
+    "Booking": {
+      "type": "object",
+      "properties": {
+        "bookingSource": {
+          "type": "string"
+        },
+        "calculatedAirCharges": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CalculatedAirCharge"
+          }
+        },
+        "charges": {
+          "$ref": "#/definitions/Charges"
+        },
+        "dateBookedLocal": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "itinSourceId": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "itinSourceName": {
+          "type": "string"
+        },
+        "lastTicketDateUtc": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "railPayments": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RailPayment"
+          }
+        },
+        "railQuotes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RailPayment"
+          }
+        },
+        "recordLocator": {
+          "type": "string"
+        },
+        "remarks": {
+          "$ref": "#/definitions/Remarks"
+        },
+        "segments": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Segment"
+          }
+        }
+      }
+    },
+    "CalculatedAirCharge": {
+      "type": "object",
+      "properties": {
+        "netFare": {
+          "type": "number"
+        },
+        "totalFareCurrency": {
+          "type": "string"
+        }
+      }
+    },
+    "Charges": {
+      "type": "object",
+      "properties": {
+        "fixed": {
+          "$ref": "#/definitions/Fixed"
+        }
+      }
+    },
+    "ClientSetting": {
+      "type": "object",
+      "properties": {
+        "enableCompanyNui": {
+          "type": "boolean"
+        },
+        "entityId": {
+          "type": "string"
+        }
+      }
+    },
+    "Comment": {
+      "type": "object",
+      "properties": {
+        "author": {
+          "description": "Comment's author (Create by)",
+          "$ref": "#/definitions/Employee"
+        },
+        "creationDateTime": {
+          "type": "string",
+          "description": "Creation date of the comment (in the format YYYY-MM-DD HH:mm:ss.SSS)"
```

Comment:
Is it not in ISO format ?

Reply:
It seems not. ISO format https://en.wikipedia.org/wiki/ISO_8601